frontend/server/src/Controllers/User.php

```diff
@@ -2433,6 +2434,35 @@ public static function getFullRankByProblemsSolved(
         ];
     }
 
+    /**
+     * Gets the best users of the current month
+     *
+     * @return list<CoderOfTheMonth>
+     */
+    public static function getTopCodersOfTheMonth(
+        int $rowCount
+    ): ?array {
```

Comment:
```suggestion
    ): array {
```

con eso ya debería pasar.